lib/insights/api/common/rbac/service.rb

```diff
@@ -15,7 +15,7 @@ def self.call(klass)
 
           def self.paginate(obj, method, pagination_options, *method_args)
             Enumerator.new do |enum|
-              opts = { :limit => 10, :offset => 0 }.merge(pagination_options)
+              opts = { :limit => 1000, :offset => 0 }.merge(pagination_options)
```

Comment:
I would agree that 10 is too small but something tells me using this max value is not correct either.  While it will reduce the number of times we have to call the RBAC service it will take time to respond and parse the results.

It would be useful to discuss the bigger picture of what problem we are trying to solve.  How is this call being used and are there other possible solutions.

cc @syncrou